spec/services/form_response_spec.rb

```diff
@@ -177,6 +177,25 @@
         expect(combined_response.to_h).to eq response_hash
       end
 
+      context 'with error detail symbol defined as type option on error' do
+        it 'returns a hash with success, errors, and error_details keys' do
+          errors = ActiveModel::Errors.new(build_stubbed(:user))
+          errors.add(:email_language, 'Language cannot be blank', type: :blank)
```

Comment:
I may be misinterpreting, but in trying to refamiliarize myself with the errors API ([source](https://github.com/rails/rails/blob/main/activemodel/lib/active_model/errors.rb), [docs](https://api.rubyonrails.org/v7.1.1/classes/ActiveModel/Errors.html)), I think this usage isn't actually "standard", and the sort of thing we're trying to enforce with `type` is probably better articulated by always specifying type as the second argument, and providing a `message` option instead.

```rb
# Instead of...
errors.add(:email_language, 'Language cannot be blank', type: :blank)

# ...do this:
errors.add(:email_language, :blank, message: 'Language cannot be blank')
```

However, at this point, we already have a linter to enforce the former, and it's pretty widespread throughout the application. Supporting it this way gets us to the same net result, but we may also want to consider migrating to the more "standard" option.